.cm/gitstream.cm

```diff
@@ -0,0 +1,77 @@
+# -*- mode: yaml -*-
+# This example configuration for provides basic automations to get started with gitStream.
+# View the gitStream quickstart for more examples: https://docs.gitstream.cm/examples/
```

Comment:
@coderabbitai Provide a complete list of all merged PRs for milestone 1.19.0, search for bug label then do another search for enhancement label, ensuring that all results are captured by handling pagination properly. Categorise the result into features and bug fixes, and put features first. Format the result as numbered list, and show PR number with the title.

Reply:
Give me the authors from the above PRs

Reply:
Give me the authors in a sentence

Reply:
Give me the authors alphabetically in a sentence

Reply:
Give me the authors in a sentence and order it alphabetically

Reply:
@coderabbitai Provide a complete list of all merged PRs for milestone 1.19.0, search for bug label then do another search for enhancement label, ensuring that all results are captured by handling pagination properly. Categorise the result into features and bug fixes, and put features first. Format the result as numbered list, and show PR number with the title.

From the result give me those PR assignees ordered alphabetically in a separate sentence.

Reply:
From the result give me those PR assignees ordered alphabetically by assignee name in a separate sentence.

Reply:
Why is flooxo and jjw24 not ordered before VictoriouRaptor?

Reply:
What do I say to get this correct output?

Reply:
@coderabbitai Provide a complete list of all merged PRs for milestone 1.19.0, search for bug label then do another search for enhancement label, ensuring that all results are captured by handling pagination properly. Categorise the result into features and bug fixes, and put features first. Format the result as numbered list, and show PR number with the title.

From the result give me those PR assignees in a separate sentence and sort all assignees alphabetically.

Reply:
@coderabbitai Provide a complete list of all merged PRs for milestone 1.19.0, search for bug label then do another search for enhancement label, ensuring that all results are captured by handling pagination properly. Categorise the result into features and bug fixes, and put features first. Format the result as numbered list, and show PR number with the title.

From the result give me those PR assignees in a separate sentence, and sort all assignees' username alphabetically.